helm/app-operator/templates/configmap.yaml

```diff
@@ -16,6 +16,8 @@ data:
     service:
       app:
         unique: {{ include "resource.app.unique" . }}
+      chart:
+        webhookBaseURL: 'http://{{ include "resource.default.name" . }}.{{ include "resource.default.namespace" . }}:{{ .Values.port }}'
```

Comment:
Build a webhook base URL from flags, and use it from resource-level is more efficient.